### DISCUSSION TITLE
Discord Bot - Custom Branding Icon

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#550] [Suggestions]
To replace the GSA icon on the GSA discord bot with an icon uploaded for community's own branding. Would this be a possible feature? I've seen bots like Sapphire allow for premium users to upload their own icon to replace their standard one on the Sapphire bot.

**Comment by mvaessen** [comment]
Hi,

Afaik is this only possible for bots that are hosted individually. You can only set up 1 image per bot.

I'm open to specific work arounds if there are any, but probably requires hosting the bot for each community separately, which isn't something im planning on doing atm.
